src/umap.cpp

```diff
@@ -57,8 +57,7 @@ class _umap {
         void* segment_address;
         size_t segment_length;
         int backingfile_fd;
```

Comment:
Yes, I keep it only because I didn't want to touch evict_page. Will do it later.